File: src/main/java/com/taw/polybank/controller/LoginController.java
package com.taw.polybank.controller;

import com.taw.polybank.dao.*;
import com.taw.polybank.entity.*;
import jakarta.servlet.http.HttpSession;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.security.crypto.argon2.Argon2PasswordEncoder;
import org.springframework.security.crypto.bcrypt.BCrypt;
import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.ModelAttribute;
import org.springframework.web.bind.annotation.PostMapping;

import java.nio.charset.Charset;
import java.security.SecureRandom;
import java.sql.Timestamp;
import java.time.Instant;
import java.util.List;
import java.util.Random;

@Controller
public class LoginController {


    @GetMapping("/")
    public String doShowIndex(){
        return "index";
    }

    @GetMapping("/login")
    public String doLogin()
    {
        return ("login");
    }

}

File: src/main/java/com/taw/polybank/controller/LoginController.kt
package com.taw.polybank.controller

import com.taw.polybank.dao.AuthorizedAccountRepository
import com.taw.polybank.dao.BankAccountRepository
import com.taw.polybank.dao.ClientRepository
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

/**
 * @author José Manuel Sánchez Rico
 */
@Controller
class LoginController {
    @Autowired
    private val authorizedAccountRepository: AuthorizedAccountRepository? = null

    @Autowired
    private val clientRepository: ClientRepository? = null

    @Autowired
    private val bankAccountRepository: BankAccountRepository? = null

    @GetMapping("/")
    fun doShowIndex(): String = "index"

    @GetMapping("/login")
    fun doLogin(): String = ("login")
}
